pkg/client/connector/userd_trafficmgr/install.go

```diff
@@ -72,6 +73,7 @@ func (ki *installer) removeManagerAndAgents(c context.Context, agentsOnly bool,
 				return
 			}
 			if _, ok := ann[annTelepresenceActions]; !ok {
+				webhookAgentChannel <- agent
```

Comment:
```suggestion
				webhookAgents = append(webhookAgents, agent)
```

Reply:
This is a concurrent write? Is this thread-safe?